nixos/modules/rename.nix

```diff
@@ -63,7 +63,7 @@ with lib;
     (mkRenamedOptionModule [ "services" "tarsnap" "config" ] [ "services" "tarsnap" "archives" ])
 
     # ibus
-    (mkRenamedOptionModule [ "programs" "ibus" "plugins" ] [ "i18n" "inputMethod" "ibus" "engines" ])
+    (mkRenamedOptionModule [ "programs" "ibus" "plugins" ] [ "i18n" "input-method" "ibus" "engines" ])
```

Comment:
@joachifm Should I make a new PR to fix that or can it be fixed by amending the commit?